accountCredentials.md

```diff
@@ -17,6 +10,40 @@ Specific information for finding your credentials can be found [here](https://su
 | `password`                             | Your **password** to *log in* to the [Bandwidth Dashboard](https://dashboard.bandwidth.com)                                                                                                                                                                                                          |  `correct-horse-battery-staple`       |
 | `accountId`                            | Your unique account **id**.  The `accountId` is used as part of the url to make API requests. <br> Ex: `https://dashboard.bandwidth.com/api/accounts/{accountId}/`                                                                                                                                   |  `920012`                             |
 | `subAccountId` <br> -or- <br> `siteId` | Any **id** for any sub-account.  The `subAccountId` is used as part of the url to make some API requests. <br> Ex: `https://dashboard.bandwidth.com/api/accounts/{accountId}/sites/{sideId}` <br><br> ℹ️ *Sub-accounts* are also known as *sites* <br> ℹ️ An *account* can have multiple *sub-accounts*|  `13606`                              |
-| `userId`                               | Your Bandwidth Account ID. The `userId` is used as part of the url to make API requests. This may be the same value as your accountId. <br> EX: `https://messaging.bandwidth.com/api/v2/users/{userId}/messages` <br><br> ⚠️ This **IS NOT** the same as your log in! ⚠️                                                                                |  `u-sd89g93epa2nb4euazd83kv8` OR `920012` |
+| `userId`                               | Your Bandwidth Account ID. The `userId` is used as part of the url to make API requests. This may be the same value as `accountId`. <br> EX: `https://messaging.bandwidth.com/api/v2/users/{userId}/messages` <br><br> ⚠️ This **IS NOT** the same as your log in! ⚠️                                                                                |  `u-sd89g93epa2nb4euazd83kv8` OR `920012` |
```

Comment:
No need to use `userId` anymore; in a v2 world it's only account id

Reply:
Done 👍